### DISCUSSION TITLE
Welcome to raceintospace Discussions!

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#499] [General]
## 👋 Welcome!
  It looks like GitHub has started a Discussions feature!  This would be much better than using Issues to talk things over, as we've done in the past.  (The rest of this is, of course, boilerplate.)

We in the (admittedly small) RIS group hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mintdog58** [comment]
i happen to own an original copy of this game and i have to tell you you guys have done an outstanding !!!! job porting this to open source

**Comment by mintdog58** [comment]
[buzz.pdf](https://github.com/raceintospace/raceintospace/files/5761603/buzz.pdf)
hers the manual

**Reply by mintdog58** [reply]
i have beatin this game landed on the moon with the lem in june 69 this game is a teacher as werll as a sim

**Reply by mintdog58** [reply]
bets all arond pc game ever mader i love it

**Reply by mintdog58** [reply]
when i saw this in the repos i freaked !!!!!!!!!!

**Comment by mintdog58** [comment]
you gotta be gutsy to beat this game take big risks

**Reply by peyre** [reply]
It's much easier to play the game than it used to be, and much easier for me to play, since I know it so well nowadays...yet it's still a challenge to beat the computer, even when I'm at level 1 and it's at 3.  I lose a lot of games.

**Comment by mintdog58** [comment]
i fly the mercury manned miision with the capsule at only 50 percent reliabilty sometimes it works

**Comment by peyre** [comment]
Hi Mintdog!  I'm glad you've found us, and thanks for the kind words.  What do you think of the changes we've made over the original?

That looks like the manual from the old Home of the Underdogs site.  Doesn't hurt to have it!  I made a higher-res PDF of the manual (and the Player's Aid Card) from better-quality scans of the paper manual, and with typos corrected.  You can download it from The Docking Module and our old Sourceforge page.  We also have an up-to-date manual for the game in its current state, in the Code section under **doc**.

We're hoping to put out a new release soon, version 2.0; unfortunately I can't give you a good ETA for it.

You wouldn't happen to be a programmer, would you?

**Comment by peyre** [comment]
Incidentally Mintdog, you don't have to be a programmer to make a contribution.  What do you think of my suggestion at #354?

**Comment by sflynn850** [comment]
Hey, this is super cool! I loved playing this game years ago, and I'm excited to clone the code down and build it :)

**Reply by peyre** [reply]
Hey, if you have some coding experience (esp. with C++), you might consider joining the team.  We still have a number of improvements we'd like to make.

**Reply by sflynn850** [reply]
Hey that sounds like a lot of fun! I'm a senior dev professionally and I could use to brush up on my c++...

**Reply by peyre** [reply]
That sounds ideal.  Welcome aboard!  Our goal is to perfect the game as is, so please don't make any big changes without consulting the group.  Any chance you might feel like looking into #209?

**Comment by sflynn850** [comment]
That sounds great - I loved this game so much and can't believe I get to
work on it.

Sure! I'll have a look at #209 this week and see what's up.

**Reply by peyre** [reply]
Incidentally, if you've had trouble getting the game to compile, I've just updated the Linux instructions.  If you're trying in Windows, that itself might be your problem - I don't think it compiles in Windows, and I can't figure out why (#498).
